Makefile

```diff
@@ -49,15 +49,15 @@ MEDIAPIPE_DISABLE ?= 0
 # NOTE: when changing any value below, you'll need to adjust WORKSPACE file by hand:
 #         - uncomment source build section, comment binary section
 #         - adjust binary version path - version variable is not passed to WORKSPACE file!
-OV_SOURCE_BRANCH ?= releases/2023/0
-OV_CONTRIB_BRANCH ?= releases/2023/0
+OV_SOURCE_BRANCH ?= 1588a332173a8401da4e34e8f6f42c693dc4dcee
+OV_CONTRIB_BRANCH ?= master
```

Comment:
Why we don't use SHA ? `master` on contrib can change (and stop working) anytime.